templates/tab/js/default/src/components/sample/PersonCardGraphToolkit.jsx

```diff
@@ -0,0 +1,28 @@
+import { PersonCard } from "@microsoft/mgt-react";
+import { useTeamsFx } from "./lib/useTeamsFx";
+
+export function PersonCardGraphToolkit(props) {
+  const { themeString } = useTeamsFx();
+
+  return (
+    <div className="section-margin">
+      <p>This example uses Graph Toolkit's
+        <a href="https://docs.microsoft.com/en-us/graph/toolkit/components/person-card" target="_blank" rel="noreferrer">person card component</a>with
+        <a href="https://github.com/microsoftgraph/microsoft-graph-toolkit/tree/next/teamsfx/packages/providers/mgt-teamsfx-provider" target="_blank" rel="noreferrer">TeamsFx provider</a>to show person card.
+      </p>      
+      <pre>{`const provider = new TeamsFxProvider(credential, scope); \nProviders.globalProvider = provider; \nProviders.globalProvider.setState(ProviderState.SignedIn);`}</pre>
+
+      {!props.loading && props.error && (
+        <div className="error">
+          Failed to read your profile. Please try again later. <br /> Details: {props.error.toString()}
+        </div>
+      )}
+      {!props.loading && !props.error && (
+        <div className={
+          themeString === "default" ? 'mgt-light' : 'mgt-dark'
+        }>
+          <PersonCard personQuery="me" isExpanded={false} ></PersonCard>
```

Comment:
![1645673257(1)](https://user-images.githubusercontent.com/26134943/155452458-fd2ae289-4192-4f00-ba1c-f8ef5968c4f3.png)
Seems this element show a white line when not authorized. Can we fix this?

Reply:
Fixed